Backend/WebApi-PointMap/Controllers/UserController.cs

```diff
@@ -81,10 +91,7 @@ public async Task<IHttpActionResult> DeleteFromSSO()
 
                     var _userManager = new UserManagementManager(_db);
                     var user = _userManager.GetUser(session.UserId);
```

Comment:
Since for this function you're expecting either a user or null if user does not exist, then there is no need to throw an error, instead can just check if return is null then return response. Vong pointed that that